dependencies.gradle

```diff
@@ -0,0 +1,28 @@
+ext.complexify = [
+    "minSdk"     : 21,
+    "compileSdk" : 30,
+    "targetSdk"  : 30,
+    "versionName": "3.0.0",
+    "versionCode": 3 * 100 * 100 + 0 * 100 + 0
+]
+
+ext.versions = [
+    "appcompat"      : "1.2.0",
+    "bintray"        : "1.8.5",
+    "gradle"         : "4.1.1",
+    "kotlin"         : "1.4.21",
+    "maven"          : "1.5"
+]
+
+ext.packages = [
+    "androidx"       : [
+        "appcompat": "androidx.appcompat:appcompat:${versions.appcompat}"
+    ],
+    "bintray"        : "com.jfrog.bintray.gradle:gradle-bintray-plugin:${versions.bintray}",
```

Comment:
Remove this, it can be deployed to Bintray without their bloated soon to die plugin too.